pkg/document/time/version_vector.go

```diff
@@ -81,6 +96,7 @@ func (v VersionVector) Marshal() string {
 
 // AfterOrEqual returns whether this VersionVector is causally after or equal
 // the given VersionVector.
+// TODO(JOOHOJANG) check below function is necessary
```

Comment:
Did we need this function? If we didn't need it, I think we could delete it.

```suggestion
// TODO(JOOHOJANG): check below function is necessary
```